src/service/url-expander/expander.js

```diff
@@ -34,36 +44,45 @@ export class Expander {
   /**
    * Link this instance of parser to the calling UrlReplacment
    * @param {!../variable-source.VariableSource|null} variableSource the keywords to replace
+   * @param {!Object<string, *>=} opt_bindings additional one-off bindings
+   * @param {!Object<string, *>=} opt_collectVars Object passed in to collect
```

Comment:
add a TODO to remove this `opt_collectVars`. an output object reference passed in constructor is error-prone.

Reply:
Done.